packages/react-data-grid/src/Cell.js

```diff
@@ -297,6 +303,14 @@ const Cell = React.createClass({
     }
   },
 
+  removeScroll() {
+    let node = ReactDOM.findDOMNode(this);
```

Comment:
😱 isn't there another way to do this? when using react mutating the DOM directly is dangerous and not adviced at all